collectors/CMPCollector.js

```diff
@@ -32,28 +47,60 @@ function isIgnoredEvalError(e) {
     );
 }
 
-class CMPCollector extends BaseCollector {
+// TODO: check for false positive detections per pattern
+const DETECT_PATTERNS = [
+    /accept cookies/ig,
+    /accept all/ig,
+    /reject all/ig,
+    /only necessary cookies/ig, // "only necessary" is probably too broad
+    /by clicking.*(accept|agree|allow)/ig,
+    /by continuing/ig,
+    /we (use|serve)( optional)? cookies/ig,
+    /we are using cookies/ig,
+    /use of cookies/ig,
+    /(this|our) (web)?site.*cookies/ig,
+    /cookies (and|or) .* technologies/ig,
+    /such as cookies/ig,
+    /read more about.*cookies/ig,
+    /consent to.*cookies/ig,
+    /we and our partners.*cookies/ig,
+    /we.*store.*information.*such as.*cookies/ig,
```

Comment:
looking at nationalgeographic.com, this pattern seems a bit too broad as it ends up matching:

>           "WED\",\"PERFORMANCE\":\"ALLOWED\",\"STRICTLY_NECESSARY\":\"ALLOWED\",\"TARGETING_ADVERTISING\":\"OPT_IN\"},\"marketingPermissions\":{\"3RD_PARTY\":\"OPT_IN\",\"EMAIL_NEWSLETTER\":\"OPT_IN\",\"FOB\":\"OPT_IN\",\"LOB\":\"OPT_IN\"}},\"",

Note `"WED\"`. Perhaps we should replace `.*` with sth like`\s.*\s`?

Reply:
it's not ideal, but `.*` is pretty generic anyway. Chances are, "we" occurred somewhere else before this snippet, so adding `\s` will probably not help much on scale. When I was testing manually, these patterns were giving little false positives, so I'd leave it as is for now.